addons/main/stringtable.xml

```diff
@@ -401,6 +401,34 @@
             <English>Allow self revive</English>
             <Polish>Zezwól na samodzielne podniesienie</Polish>
         </Key>
+        <Key ID="STR_diw_armor_plates_main_visibleBleedoutTimer">
+            <English>Show bleedout timer to</English>
+            <Polish>Komu powinno się pokazać odliczanie do śmierci?</Polish>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_visibleBleedoutTimer_desc">
+            <English>Applicable players will see a countdown over the wounded icon displaying the time until death.</English>
+            <Polish>Wybrani gracze zobaczą odliczanie nad ikoną rannych wyświetlające czas do śmierci.</Polish>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_visibleBleedoutTimer_nobody">
+            <English>Nobody</English>
+            <Polish>Nikt</Polish>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_visibleBleedoutTimer_medics">
+            <English>Medics</English>
+            <Polish>Medycy</Polish>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_visibleBleedoutTimer_everyone">
```

Comment:
Used `str_a3_cfgvehicles_modulesector_f_arguments_taskowner_values_everyone_0` instead of making a new string table entry

There is a tool to search for them
https://brettmayson.github.io/stringtables/